.ci/create-release-github.sh

```diff
@@ -1,5 +1,19 @@
 #!/bin/bash
 
+# Copyright The OpenTelemetry Authors
```

Comment:
The header is already defined in `header.txt`. The operator already uses golang-ci to verify the license header is present on the files. 

Could we configure the golang-ci to check the target allocator as well? It seems that the golang-ci is not checking the target allocator, we should configure it to check the whole repo.

Reply:
I will work on fixing golang-ci to check the target allocator files instead of using the make target.  What about `.sh` files - should we keep the target for those?

Reply:
I am with not adding the headers to the .sh files. Those files are only to support CI or local development.